kboparai1/ex11/sd01.py

```diff
@@ -0,0 +1,6 @@
+# Pythons raw_input takes in a value that is entered by the user. 
+#
+# raw_input([prompt])
+#
+# If the prompt argument is present, it is written to standard output without a trailing newline. The function then reads a line from input, converts it to a string (stripping a trailing newline), and returns that. When EOF is read, EOFError is raised. Example:
```

Comment:
I like you looking it up, if possible, type it again in your own words instead of copy and pasting. Forces you to analyze a bit more and more likely to remember things.